src/Button.js

```diff
@@ -0,0 +1,15 @@
+import Field from './Field'
+
+function Button({label, disabled, children}) {
+    return (
+      <div className='flex__row margin__top__24 margin__bottom__24'>
+        <button disabled={disabled} className='button__signup'>{label}</button>
+        <div className='flex__row center__vertical'>
```

Comment:
This section shouldn't be inside the Button component, as it's not related, and makes this component not reusable.